assets/js/util/is-url-using-https.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Utility function related to checking if a given URL uses HTTPS.
```

Comment:
I added this function to the global utils, so it can be reused between modules, I did not remove it from RRM module, or replace it's usage, in order not to extend the scope here, and will notify team M that they can replace it there in their folow up issue that will include HTTPS on the setup page